avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -1532,7 +1534,7 @@ private Schema toConnectSchema(org.apache.avro.Schema schema, boolean forceOptio
     }
 
     JsonNode parameters = schema.getJsonProp(CONNECT_PARAMETERS_PROP);
-    if (parameters != null) {
+    if (connectMetaData && parameters != null) {
```

Comment:
there seem to be other attributes of the schema that are set based on `connectMetaData` (for example: version and name). Do we want to make that behavior consistent between these two methods?

Reply:
That's a very good question.  I was trying to be a bit conservative, but let me look at it a bit more.  Perhaps they should be as symmetrical as possible, as you suggest.

Reply:
@wicknicks , I added a comment as to why the current parameters are the only ones omitted for now.